Change to login and signup

Royd117 opened the pull request:
Changes to email address in signup and login acording to model

Royd117 committed:
Read user

Read user first itteration

Royd117 committed:
Change to login and signup

Changes to email address in signup and login acording to model

Royd117 committed:
Merge branch 'QA' into feature/userInfo

Royd117 committed:
Change in previous functions

These functions had camelcase instead of using dashes and changed user for User in read-user controller

Royd117 committed:
Merge branch 'QA' into feature/userInfo

Royd117 committed:
Merge branch 'QA' into feature/models

RazielHR reviewed (dismissed):
no lo lei pero confio en montu

Royd117 committed:
Merge branch 'feature/userInfo' into feature/models

Royd117 committed:
Changes to User and dependencies

Changes to user model and dependencies because of postgresSQL reserved word User

Royd117 committed:
Refactor User, Role and Database setup.

User and Role changed to DefaultUser and UserRole due conflicts with protected keywords and database setup(Schema and Seeder)
